public/js/components/ButtonSettings.vue

```diff
@@ -27,20 +27,30 @@
       </select>
     </label>
 
-    <label v-if="tile.callType === 'method'">
+    <label for="deviceMethod" v-if="tile.callType === 'method'">
```

Comment:
```suggestion
    <label v-if="tile.callType === 'method'" for="deviceMethod">
```

(For the same reason as in my previous comment about ESLint.)

Reply:
Reverted 👍